RocketChat/Rocket.Chat
# feat: Verification check filter for Omnichannel rooms

## Proposed changes (including videos or screenshots)
Implemented a verification check filter to sort chats based on the room's verification status.

https://github.com/RocketChat/Rocket.Chat/assets/67470541/7ed12fca-898c-402e-bbb9-d73e4ffff8d6






## Issue(s)


## Steps to test or reproduce
1. Go to /omnichannel/current
2. Filter the chats according to the categories available in the "Verification Check" dropdown section.


## Further comments
